packages/block-editor/src/components/iframe/index.js

```diff
@@ -203,11 +203,7 @@ function Iframe(
 			return true;
 		}
 
-		if ( setDocumentIfReady() ) {
-			return;
-		}
-
-		// Document is not immediately loaded in Firefox.
+		// Document set with srcDoc is not immediately ready.
 		node.addEventListener( 'load', () => {
 			setDocumentIfReady();
 		} );
```

Comment:
This can be done without the arrow function:
```js
node.addEventListener( 'load', setDocumentIfReady );
```
`setDocumentIfReady` will receive an `event` parameter which it doesn't use anyway.

Also, to tidy things up, the effect callback should return a cleanup function:
```js
return () => node.removeEventListener( 'load', setDocumentIfReady );
```

Reply:
done and done :)